kiwi/builder/live.py

```diff
@@ -359,6 +347,30 @@ def create(self) -> Result:
         )
         return self.result
 
+    def _bootloader_instance(self) -> BootLoaderConfigBase:
```

Comment:
Maybe use a name that suggests that an instance is explicitly created and this is not just a getter?
```suggestion
    def _create_bootloader_instance(self) -> BootLoaderConfigBase:
```

Reply:
yep done